packages/semantic-release-alt-publish-dir/index.js

```diff
@@ -38,18 +38,16 @@ async function prepare(
     return;
   }
 
+  logger.log(`Building alternative root package.json`);
+  await createAltPublishDir({ outDir: pkgRoot });
+
   const pkgPath = join(cwd, './package.json');
   // eslint-disable-next-line import/no-dynamic-require
   const pkg = require(pkgPath);
 
-  logger.log('Updating root package.json file to version %s', version);
-
   // update the root package since @semantic-release/npm won't;
```

Comment:
`@semantic-release/npm` now uses `npm version`, which will fail when it sees that the `package.json` in `pkgRoot` has already been version-bumped